src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/histogram.js

```diff
@@ -0,0 +1,533 @@
+loadScript('https://d3js.org/d3.v4.min.js');
+loadStyle('https://fontlibrary.org/face/dejavu-sans-mono')
```

Comment:
When these styles / scripts are being loaded? The thing is that they should be loaded ON FIRST USAGE of the histogram, not earlier, as our GUI startup time is huge already. I believe that now they are loaded when gui starts, as we are parsing the JS file and we are caching the class definition, right? If I'm right, we need to fix it probably inside of constructor of the visualization. I would not think of any generic mechanism for it for now, let's see other visualizarions and then lets generalize it.

*EDIT*

In the future, I would allow for ES6 import syntax here and I would handle it in a special way (import on first usage);

Reply:
It actually seems to be a bit more tricky because if we want to use it in the constructor, we need for the script to be loaded via `onload` on the script, which is probably the more correct way to do this anyway. But that requires us to start making things async/callback based, which makes this a bit more difficult. I would suggest moving this to another issue because it might require some more significant changes to all visualizations. (Unless there is an easier way that I don't know about)

Reply:
As we discussed, let's do it after this PR as the startup time in GUI is very important and right now its already ugly. Please discuss it with Sylwia as well.

Reply:
Issue added: https://github.com/enso-org/ide/issues/985